docs/reference/mapping/runtime.asciidoc

```diff
@@ -765,21 +847,132 @@ and determine which fields to index.
     "hits" : [
       {
         "_index" : "my-index",
-        "_id" : "oWs5KXYB-XyJbifr9mrz",
+        "_id" : "hdEhyncBRSB6iD-PoBqe",
         "_score" : 1.0,
         "_source" : {
-          "@timestamp" : "2020-06-21T15:00:01-05:00",
-          "message" : "211.11.9.0 - - [2020-06-21T15:00:01-05:00] \"GET /english/index.html HTTP/1.0\" 304 0"
+          "timestamp" : "2020-04-30T14:31:27-05:00",
+          "message" : "252.0.0.0 - - [30/Apr/2020:14:31:27 -0500] \"GET /images/hm_bg.jpg HTTP/1.0\" 200 24736"
+        }
+      },
+      {
+        "_index" : "my-index",
+        "_id" : "htEhyncBRSB6iD-PoBqe",
+        "_score" : 1.0,
+        "_source" : {
+          "timestamp" : "2020-04-30T14:31:28-05:00",
+          "message" : "not a valid apache log"
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/\.\.\./"took" : $body.took,"timed_out" : $body.timed_out,"_shards" : $body._shards,/]
+// TESTRESPONSE[s/"_id" : "hdEhyncBRSB6iD-PoBqe"/"_id": $body.hits.hits.0._id/]
+// TESTRESPONSE[s/"_id" : "htEhyncBRSB6iD-PoBqe"/"_id": $body.hits.hits.1._id/]
+
+[[runtime-examples-dissect]]
+==== Define a runtime field with a dissect pattern
+<<dissect-processor,Dissect patterns>> are similar to grok patterns, but they
+don't accept regular expressions. This difference allows dissect’s syntax to be
```

Comment:
I think something more along the lines of "if you don't need the power of regular expressions you can use `dissect` patterns. They match on fixed delimiters but are generally faster that dissect. Parsing the apache logs from above looks like...."

Reply:
Just a little more positive about the advantages while still pointing out the differences.